### DISCUSSION TITLE
Staff propriety in clefs

### DISCUSSION THREAD

**Discussion by fosfrancesco** [CPJKU/partitura#123] [Ideas]
There are several elements in partitura that have a staff-like attribute. In particular:
- all subclasses of GenericNote
- all subclasses of Direction
- Words
- Clef

Such attribute is always called '''staff''' except in Clefs where it is called '''number'''.
I guess the initial reason for that was that it is called like that in musicxml. Can I change that? This is not backward compatible and will create problems in all old code using this propriety. But I think this will not be a lot of code and now it is a good time to do it.

**Comment by sildater** [comment]
sounds good to me. make sure to change it in the relevant i/o functions too.

**Comment by manoskary** [comment]
+1 for fixing this issue. In either case, I think the next release of partitura should be a proper one then, i.e. version 1.0.0
